packages/wonder-blocks-popover/components/popover.md

```diff
@@ -0,0 +1,265 @@
+#### Example: Opening a popover using a trigger element
```

Comment:
It may be worth mentioning at the top before the examples what combinations of props aren't allowed.

Reply:
Mentioned that on each variant example. Do you think is it easier to read at the top instead?

Reply:
Not sure.  Let's give it a try and see if people are confused or not.